packages/ui/src/components/Button/button.tsx

```diff
@@ -28,13 +30,24 @@ const Button = React.forwardRef<HTMLButtonElement, ButtonProps>((props, ref) =>
         ? <span className={`${BUTTON_CLASS_PREFIX}-icon`}>{icon}</span>
         : null
 
+    const handleClick = useCallback((event) => {
+        if (isString(children)) {
+            sendAnalyticsClickEvent('Button', { value: children })
```

Comment:
What about buttons with icons and text? Or buttons with just icon as a children?

Reply:
Icon are passed with Icon property

Reply:
And for button without a text, I don't really know, how to generate name dynamically